blueprints/secops/age/ops2deb.yml

```diff
@@ -0,0 +1,20 @@
+name: age
+matrix:
+  versions:
+    - 1.0.0
+    - 1.1.0
+    - 1.1.1
+  architectures:
+    - amd64
+    - arm64
+homepage: https://age-encryption.org/
+summary: age is a simple, modern and secure file encryption tool, format, and Go library
```

Comment:
The convention in debian packages is to not repeat the name of the project at the start of the summary: `simple, modern and secure encryption tool`

Reply:
thx fixed